.github/workflows/upload-coverage-report.yml

```diff
@@ -0,0 +1,53 @@
+name: Upload Coverage Report
+
+on:
+  workflow_run:
+    workflows:
+      - LLPC Docker CI
```

Comment:
Later in the file we specify `workflow: check-amdllpc-docker.yml`. Would it be possible to use the filename instead of the textual name here as well?

Reply:
No, this is a limitation of GitHub. The other action is a custom action that handles it specially.